webapp/src/component/layout/TopBanner/TopBanner.tsx

```diff
@@ -17,9 +17,15 @@ const StyledContainer = styled('div')`
   display: grid;
   grid-template-columns: 50px 1fr 50px;
   width: 100%;
-  background: ${({ theme }) => theme.palette.topBanner.background};
+  background: ${({ theme }) =>
+    useIsEmailVerified()
```

Comment:
This is not a good way how to use hooks. Use it in the component itself and only apply class to change styles.

https://react.dev/reference/rules/rules-of-hooks